OpenOversight/app/main/views.py

```diff
@@ -599,11 +599,93 @@ def add_unit():
         db.session.add(unit)
         db.session.commit()
         flash('New unit {} added to OpenOversight'.format(unit.descrip))
-        return redirect(url_for('main.get_started_labeling'))
+        return redirect(url_for('main.view_unit.{}'.format(unit.id)))
     else:
         return render_template('add_unit.html', form=form)
 
 
+def get_form_and_choices(department_id, race=[], gender=[], rank=[], min_age='16', max_age='100', name=None, badge=None, unique_internal_identifier=None):
```

Comment:
this whole function is from https://github.com/lucyparsons/OpenOversight/blob/develop/OpenOversight/app/main/views.py#L457-L511 - in a future merge I can have that function use this helper function as well? Was trying to keep code touches low